### DISCUSSION TITLE
Setting up RDIO with Proscan.

### DISCUSSION THREAD

**Discussion by bobfeser** [chuot/rdio-scanner#319] [Q&A]
I am new to rdio and trying to set it up with Proscan. I have a BCD396T and have lots of traffic that I stream to RR but I am trying to set up rdio so I can listen to select TG when I am at work. My issue that I am having is getting RDIO to ingest files. I have set up DIR watch and also have the Auto Populate checked. When I look at the logs for rdio it says dirwatch.ingest: no talkgroup, C:\Proscan\Recordings\date\

![image](https://user-images.githubusercontent.com/123333205/214603292-ebe293b5-0d4c-407d-a3f2-224160386f51.png)

I am thinking I am missing something simple and want to figure this out!

Thanks in advance for any help.

**Comment by bobfeser** [comment]
![image](https://user-images.githubusercontent.com/123333205/215206634-8214b472-8444-43e3-b4c0-76e93881e3a4.png)

**Reply by bobfeser** [reply]
And I am getting no playback of the audio.

**Reply by noamlivne** [reply]
I wonder if the "no talkgroup" is a hint.
Try to add to the Dirwatch a "Catch All"/default Talkgroup. I think it appears in the lower right part of the Dirwatch.  For example "999".
In addition, perhaps you should also need to give that Talkgroup a name in Config/System tab.
That way each recording will be under 999 but with a specific ID,  i.e "
999 Catch All 
Combined+2OPS..."

**Reply by bobfeser** [reply]
I have set up Config/System for all the different OPP TG (wide area, Regular Operations, Detachment....each on there own.) as well as MTO Enforcement and MTO Maintenance TG on there own system. I enter the Unique DB ID for each talk group and then the DEC for each TG with the matching Alpha tag. I get the feeling that the Proscan saving is not what RDIO likes for the ingest. Like I said earlier I am feeling that it is close to working and it is something simple that I am not seeing. I have also tried with my 2 RTLSDR dongles and trunking-recorder, but again it is not working. Bouncing back and forth between the two different set ups I should probably stop and just get one to work and then go to the other....lol. I get frustrated and then delete it all and start over. I have even Just set up a SYSTEM Fleetnet with a system ID of 822 and auto populate turned on. That works but none of the TG sort correctly. I will attach a couple screen shots over the next couple days when I get to work on it a little more.

This is what I am working on from RR

![image](https://user-images.githubusercontent.com/123333205/218685164-72774ccd-4072-4121-b29f-f255c2f0fbb9.png)


![image](https://user-images.githubusercontent.com/123333205/218685386-c3c09fc9-774b-4b45-9097-8c0d105d07d0.png)


Thanks for your help so far.

**Reply by noamlivne** [reply]
I wonder if you followed my tip (which was not so clear). 
Here is what I meant (but in my case it is a DSDPlus Fastlane Dirwatch):
Note the "Defualt" CatchAll as System and as Talkgroup (just in case). In addition, I added those CatchAlls as a System and as a Talkgroup, accordingly.
![image](https://user-images.githubusercontent.com/50629227/218688091-649b3656-d927-4c55-8b0f-8bb47aed011d.png)

![image](https://user-images.githubusercontent.com/50629227/218689081-f6bc3270-3565-4bb9-a3c2-a895ec974a3b.png)

**Reply by bobfeser** [reply]
Thanks for your quick reply and the pictures. I will have to look when I can at my setup. As I stated in my reply above, if I just do the "Fleetnet" with ID of 822 and auto populate I get the same TG (1,2,3) as you have listed in your second picture and it seems to be working but when I rename those to the OPP TG or MTO Maintenance TG it seems to just stop working. And I am not sure why. As I originally stated I am new to RDIO and have looked everywhere I could think of on YouTube to get a step by step setup and found nothing. I will look into this and add a couple more picture when I am on my setup. You have given me a good start!! 

Thanks

**Reply by noamlivne** [reply]
No problem.
I wonder why the 1, 2, 3 do not work.
Maybe there is some conflict?
If you try with the Default CatchAlls and it does not work, perhaps the developer can chime-in if needed.
(I have no experience with Proscan.)

**Reply by bobfeser** [reply]
You have given me some great info to keep my brain thinking about this!! Either way I will give an update in the next few days.

**Reply by bobfeser** [reply]
@noamlivne I have worked on it some this morning and I have made progress. I have RDIO playing recordings from trunking recorder...the catch is it is all as 1 TG. but it is a baby step.

![image](https://user-images.githubusercontent.com/123333205/219429568-972ff4bb-e712-4823-b344-78fc30c5c918.png)

![image](https://user-images.githubusercontent.com/123333205/219429841-1aa12d3b-5d9e-4a26-93ab-8ff78e13ac9b.png)

![image](https://user-images.githubusercontent.com/123333205/219430085-dcd3289a-01b6-4b19-9aec-884ad685141c.png)

![image](https://user-images.githubusercontent.com/123333205/219430252-3c829173-848b-45ce-a1c0-85045bf734c9.png)


I am going in the right direction.

**Comment by noamlivne** [comment]
Good News!
I would guess that is something with the Proscan's format of recordings that Rdio-Scanner does not handle well.
I hope you will succeed.

**Reply by bobfeser** [reply]
this is in  Trunking Recorder to RDIO right now off my 2 dongles. But a little progress is better than no progress!!
